### DISCUSSION TITLE
Open?

### DISCUSSION THREAD

**Discussion by pshinners-blz** [OpenAssetIO/OpenAssetIO#44] [General]
This is a personal opinion; I'm not confident in the "Open" part of the OpenAssetIO name. My cynical mind feels like this project will only be a "FoundryAssetIO". When this gets used across Foundry products this project still has tremendous value that I'm excited about. But until there is engagement with other vendors to use the Host API I remain skeptical that it will ever happen. Is there a milestone on the roadmap where this type of communication should start happening? It feels like if products from the industry don't start taking a look soon this may not a good fit for them.

An amazing goal would be adaptors that make OAIO work with existing systems. If I implement a manager with OAIO there are already built plugins for Houdini's file helper ([FS_Reader.h](https://www.sidefx.com/docs/hdk/_f_s___reader_8h_source.html)) and USD's AR2.0. This should be technically possible, but is it a goal for this project?

I'm still feeling like this needs one, or some of both of these to earn the "Open" part of its name. On the other hand, I see Khronos releasing a new "OpenWhatever" every third week of the month, so it's likely this is my personal idealism leaking through.

**Comment by foundrytom** [comment]
We'd dearly love to have other vendors adopt OAIO. There are some very early stage conversations in progress already, however there is a bit of a chicken/egg situation. Without visible support from studios, it's a hard sell to vendors, without support from vendors, it's a hard sell to studios. Fun times. 

This is one of the reasons we are moving these formative stages in OAIOs development into a public forum - we hope that discussions such as these, and open input to its technical direction will start to demonstrate life around the project, and genuine intentions, whilst we doggedly press on. We're always more than happy to open a dialog with anyone who is interested in adoption, and any vocal support for it from others would really help make the case.

> If I implement a manager with OAIO there are already built plugins for Houdini's file helper (FS_Reader.h) 

We envisioned it would be "use OAIO to resolve to an URL, use something like the Houdini API to get the data from the URL" To avoid making it too much of a monolith.

>  USD's AR2.0. 

Interop with Ar 2.0 (and the current Katana API) is in the roadmap to be addressed before we consider the core API stable.

**Comment by oumad** [comment]
I think in general the Open word isn't earned as much as it is Needed for multi vendor adoption to happen in the first place. An analogy given by a member of ASWF work groups that I like :  "You can open a door and maybe no one will through it, but the door is opened nevertheless."

You do have this chicken/egg situation as @foundrytom said, but I personally wouldn't be unhappy to see an OpenWhatever every now and then, the "open" for me is a great start, and time remains the best judge for any solution.
